### DISCUSSION TITLE
Extension support for Jetbrains products

### DISCUSSION THREAD

**Discussion by InfernalAzazel** [vue-vine/vue-vine#144] [Ideas]
一直习惯使用 idea， 目前 vue vine 插件只支持 vscode

**Comment by ShenQingchuan** [comment]
目前我们暂时没有特别多的精力来推进这个自动格式化的方案设计，因此暂时先将该问题转为了一个讨论帖，如果有同学有兴趣参与可以在这里回复。

Currently, we don't have much time to push forward this automatic formatting scheme design, so let's temporarily turn this issue into a discussion post. If anyone is interested, please feel free to reply here.
